src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -6,6 +6,14 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
```

Comment:
Can you remove this license banner? There is one already at the top of the file.